src/main/java/seedu/contax/model/person/ContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,28 @@
+package seedu.contax.model.person;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
```

Comment:
I believe this description is also wrong, this is an abstract class rather than for name.
```suggestion
 * Tests that an attribute of a {@code Person} matches any of the keywords given.
```